src/commands/game_commands/play.ts

```diff
@@ -23,22 +34,91 @@ export default class PlayCommand implements BaseCommand {
             }
             if (!gameSessions[message.guildID]) {
                 const textChannel = message.channel;
-                const gameSession = new GameSession(textChannel, voiceChannel, message.author);
+                const eliminationMode = parsedMessage.components.length >= 1 && parsedMessage.components[0].toLowerCase() === "elimination";
+                const gameOwner = message.author;
+                const lives = parsedMessage.components.length > 1 ? parseInt(parsedMessage.components[1], 10) : DEFAULT_LIVES;
+                const gameSession = new GameSession(textChannel, voiceChannel, gameOwner, eliminationMode, lives);
                 gameSessions[message.guildID] = gameSession;
-                await sendInfoMessage(message, `Game starting in #${textChannel.name} in 🔊 ${voiceChannel.name}`, "Listen to the song and type your guess!");
-                logger.info(`${getDebugContext(message)} | Game session starting`);
-                gameSession.startRound(guildPreference, message);
+
+                const lifeOrLives = lives === 1 ? "life" : "lives";
```

Comment:
Easier just to replace with "live(s)"

Reply:
I think I'll just keep it as lives, just realized the singular version (live) doesn't make sense